bibliograph/core/utils.py

```diff
@@ -135,13 +150,38 @@ def AuthorURLs(entry):
 def _braceUppercase(text):
     """ Convert uppercase letters to bibtex encoded uppercase
 
+        >>> import pretext; pretext.activate()
         >>> from bibliograph.core.utils import _braceUppercase
-        >>> _braceUppercase('foo bar')
+        >>> _braceUppercase(u'foo bar')
         'foo bar'
 
-        >>> _braceUppercase('Foo Bar')
-        '{F}oo {B}ar'
+        >>> _braceUppercase(u'foo BAr')
+        'foo {BAr}'
+
+        >>> _braceUppercase(u'foo bAAr')
+        'foo b{AA}r'
+
+        >>> _braceUppercase(u'Foo Bar')
+        '{Foo} {Bar}'
+
+        >> _braceUppercase(u'Языки Евразии')
+        '{Языки} {Евразии}'
+
     """
+    try:
```

Comment:
AFIK It's better to check the distribution instead of trying to import. Like this
https://github.com/collective/collective.handlebars/blob/master/src/collective/handlebars/browser/views.py#L15
I think it was documented somewhere but could not find it.